R/calculate_stats.R

```diff
@@ -31,13 +44,25 @@
 calculate_stats <- function(seasons = nflreadr::most_recent_season(),
                             summary_level = c("season", "week"),
                             stat_type = c("player", "team"),
-                            season_type = c("REG", "POST", "REG+POST")){
+                            season_type = c("REG", "POST", "REG+POST"),
+                            pbp = NULL){
```

Comment:
since we're validating pbp in every scenario does it make sense to do something like 
```suggestion
                            pbp = load_pbp(seasons)){
```
instead?

Reply:
> the one downside of this being that we won't be able to do the custom_pbp attribute later

That's exactly why I would like to keep the default at NULL. I feel like it won't take long until we see issues comparing stats from custom pbp with PFR or ESPN or whatever

Reply:
what's the upside in your opinion compared to pbp = NULL?